int32 ID_node
string type
string node_name

float32 server_time
float32 num_servers
float32 utiliz_tot
float32[] utiliz_array
float32[] utiliz_array_tot
float32[] queue_array
time[] time_array
time[] time_array_intervals
int32 queue_left
string info

int32 events_left

bool stop_esecution
bool statistic